src/app/presentation/components/user-header/user-header.component.ts

```diff
@@ -2,33 +2,40 @@ import { AuthenticationService } from './../../../core/services/authentication.s
 import { Component, OnInit } from '@angular/core';
 import { UserInfo } from '../../../core/models/userInfo'
 import { Router } from '@angular/router';
+import { MatDialog } from '@angular/material/dialog';
+import { ModalInvitesComponent } from '../modal-invites/modal-invites.component';
 
 @Component({
   selector: 'app-user-header',
   templateUrl: './user-header.component.html',
   styleUrls: ['./user-header.component.css'],
   providers: [AuthenticationService]
 })
+
 export class UserHeaderComponent implements OnInit {
   
   userMessageCount = '2';
```

Comment:
If you add const value please leave the comment that describe for a what reason and what it's mean